internal/asm/f64/ge_amd64.go

```diff
@@ -0,0 +1,15 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
```

Comment:
Why are these files named `ge_*`? I would not put all functions for general matrices in one file, so I think `ger_*` is more appropriate.

Reply:
I was only planning on putting the three `ge*` kernels (ger/gemv/gemm) in that file, rather than a file for each.

Reply:
Wouldn't the asm file then become too unwieldy?

Reply:
Fixed.  I didn't notice the asm file was named wrong.